contrib/guix/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
```

Comment:
Can you add `export LC_ALL=C` to make [Travis happy](https://travis-ci.org/bitcoin/bitcoin/jobs/488536092#L257).

Reply:
@dongcarl ^ ?